qa/integration/specs/logstash_to_logstash_spec.rb

```diff
@@ -0,0 +1,127 @@
+# Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
Why are we licensing this under the Elastic license? It exists outside of the top-level `x-pack` directory and covers Apache-licensed features.